go-controller/pkg/ovn/master.go

```diff
@@ -868,3 +889,8 @@ func (oc *DefaultNetworkController) createACLLoggingMeter() error {
 
 	return nil
 }
+
+// isLocalZoneNode returns true if the node is part of the local zone.
+func (oc *DefaultNetworkController) isLocalZoneNode(node *kapi.Node) bool {
```

Comment:
oh do you just want to move this to bnc in this commit (4th) itself? will cause less confusion of code moving back and forth.

Reply:
Ack